docs/api/index.md

```diff
@@ -1052,6 +1061,10 @@ When you use `test` in the top level of file, they are collected as part of the
 
   If the function throws an `Error`, the snapshot will be the error message. Otherwise, snapshot will be the value thrown by the function.
 
+  :::tip
+    To assert that an async function rejects, you must use [`rejects`](#rejects) together with [`toThrowErrorMatchingInlineSnapshot`](#toThrowerrormatchinginlinesnapshot). Otherwise Vitest will complain that it caught an unhandled error during the test run.
```

Comment:
```suggestion
    To assert that an async function rejects, you must use [`rejects`](#rejects) together with [`toThrowErrorMatchingInlineSnapshot`](#toThrowerrormatchinginlinesnapshot). Otherwise, Vitest will complain that it caught an unhandled error during the test run.
```